code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -71,10 +71,14 @@
 							span_warning("[user] punches you, but doesn't leave a dent!"), null, COMBAT_MESSAGE_RANGE, user)
 			to_chat(user, span_danger("You punch [src], but don't leave a dent!"))
 		else
-			visible_message(span_notice("[user] pets [src]."), \
-							span_notice("[user] pets you."), null, null, user)
-			to_chat(user, span_notice("You pet [src]."))
-			user.add_mood_event("pet_borg", /datum/mood_event/pet_borg)
+			if(HAS_TRAIT(src, TRAIT_QUICKREFLEXES) && (src.stat != UNCONSCIOUS))
+				visible_message(span_warning("[user] tries to pet [src], but it moves out of the way."))
+				return
+			else
+				visible_message(span_notice("[user] pets [src]."), \
+								span_notice("[user] pets you."), null, null, user)
+				to_chat(user, span_notice("You pet [src]."))
+				user.add_mood_event("pet_borg", /datum/mood_event/pet_borg)
```

Comment:
```suggestion
                        //SKYRAT EDIT BEGIN - DISALLOWED HEADPATS
			if(HAS_TRAIT(src, TRAIT_QUICKREFLEXES) && (src.stat != UNCONSCIOUS))
				visible_message(span_warning("[user] tries to pet [src], but it moves out of the way."))
				return
			else
				visible_message(span_notice("[user] pets [src]."), \
								span_notice("[user] pets you."), null, null, user)
				to_chat(user, span_notice("You pet [src]."))
				user.add_mood_event("pet_borg", /datum/mood_event/pet_borg)
			//SKYRAT EDIT END - DISALLOWED HEADPATS
```

Non modular edits need to be marked

Reply:
how'd I let this one slip by me I could've swore I got all the marks down. I'm way too tired to be coding lol, thanks